scripts/push-package.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+file="$1"
+
+prefix="$(cat $file | jq -r '.package .code')"
+version="$(cat $file | jq -r '.package .DHIS2Version' | cut -d '.' -f 1,2)"
+
+package_code="${prefix:0:4}"
+sub_package_code="${prefix:4:2}"
+
+repo=$(
+  curl -fsSL "https://api.github.com/orgs/dhis2-metadata/repos?per_page=100" |
+  jq -r --arg package_code "$package_code" '.[] | select(.name | contains($package_code)) | .name'
+)
+
+git config --global user.email "$GITHUB_EMAIL"
+git config --global user.name "$GITHUB_CREDS_USR"
+
+git clone "https://$GITHUB_CREDS_PSW@github.com/dhis2-metadata/$repo"
```

Comment:
I can't find any references to $GITHUB_CREDS_PSW, but I'm assuming it's a secret and won't be printed?

Reply:
Yes, it comes from [here](https://github.com/dhis2/metadata-package-development/blob/DEVOPS-125/Jenkinsfile#L161) - this creates two separate env vars *_PSW and *_USR ([more details here](https://www.jenkins.io/doc/book/pipeline/jenkinsfile/#usernames-and-passwords))